connexion/apis/abstract.py

```diff
@@ -32,7 +32,83 @@ def __init__(cls, name, bases, attrs):
         cls._set_jsonifier()
 
 
-class AbstractAPI(metaclass=AbstractAPIMeta):
+class AbstractSpecAPI(metaclass=AbstractAPIMeta):
```

Comment:
Can you add a docstring for clarity on what the purpose of this class is?

Reply:
Done.

We can probably remove some of these abstract API classes in the future, since they'll only be implemented for their respective middleware. For now I've kept them so it's clear which functionality is extracted from the `AbstractAPI` class.